dist/cjs/handlebars/base.js

```diff
@@ -35,15 +35,25 @@ exports.HandlebarsEnvironment = HandlebarsEnvironment;HandlebarsEnvironment.prot
       if (inverse || fn) { throw new Exception('Arg not supported with multiple helpers'); }
       Utils.extend(this.helpers, name);
     } else {
+      // CVE-2019-19919, CVE-2021-23369: Prevent prototype pollution by blocking dangerous property names
+      // Addresses Snyk vulnerabilities: 534988, 469063, 173692, 1279029, 567742
+      if (name === '__proto__' || name === 'constructor' || name === 'prototype') {
```

Comment:
I think we can replace more than 2 "||" on includes function 

['...', '....'. '.....'].includes(name)
```suggestion
      if (name === '__proto__' || name === 'constructor' || name === 'prototype') {
```

Reply:
@akrivickiy-sugarcrm I would agree with you in general but since this is a forked library and we are only fixing the security by trying to match whatever is in the library, lets leave this as it was done in the main library itself